### DISCUSSION TITLE
Desk Band Support

### DISCUSSION THREAD

**Discussion by AliShahesmaeili** [microsoft/WindowsAppSDK#2320] [Ideas]
Samples:
https://github.com/navhaxs/media-control-deskband
https://www.codeproject.com/Articles/39189/Shell-Extensibility-Explorer-Desk-Band-Tray-Notifi
https://www.ghacks.net/2021/03/07/taskbar-monitor-displays-a-deskband-to-indicate-your-computers-resource-usage/

**Comment by driver1998** [comment]
The problem is... The new XAML taskbar in Windows 11 does not support desk band anymore.

**Reply by efavry** [reply]
Would you happen to know if there is a replacement ?

**Reply by driver1998** [reply]
@efavry There "is" one, but that is to hack the taskbar and move the HWNDs inside around to make room, then put your stuff inside.

You can checkout https://github.com/zhongyang219/TrafficMonitor for an example.
